Tribler/Core/Modules/restapi/my_channel_endpoint.py

```diff
@@ -113,6 +118,52 @@ def render_PUT(self, request):
         return json.dumps({"added": channel_id})
 
 
+class MyChannelCreateTorrentEndpoint(MyChannelBaseEndpoint):
+    """
+    Create a torrent file from local files.
+
+    Example request:
+    {
+        "files": ['path/to/file.txt', 'path/to/another/file.mp4'],
+        "tracker": "http://localhost/announce",
+        "name": "Cat video",
+        "description: "Video of my cat"
```

Comment:
Comments perhaps yes, the advanced options definitely not at this point. This should be the same as the description we talked about in #2214 I suppose.